### DISCUSSION TITLE
Wifi communication drops after a few seconds of operation. Server cannot be reached.

### DISCUSSION THREAD

**Discussion by alpiliguri** [DCC-EX/CommandStation-EX#412] [Q&A]
Hi, I have mounted a EX‑CommandStation, hardware consisting of:

_Mega +WiFi R3 ATmega2560+ESP8266 (8Mb memory), USB-TTL CH340
Dual Channel DC Motor Driver Shield Expansion Board L298NH_

The command station works perfectly when connected to the PC via USB, I can control it from the browser with EX-WebThrottle, or from JMRI, via Engine Driver on a smartphone connected to the home Wifi.

The problem arises when I want to connect directly to the command station without using the PC, connecting the Engine Driver to the integrated access point (config: Access Point Mode).

**The smartphone connects to wifi, password recognized, I open Engine Driver and it automatically finds the server, it connects, I choose an address, I can drive the decoder, motor run, switch on the lights etc.... but only for 10/20 seconds!**

Then Engine Driver tells me that the server (192.168.4.1:2560) is no longer reachable, the wifi connection remains intact but the communication is interrupted.
I searched online but there aren't many articles on a similar problem, the only clue is the crowded wifi channel, I moved the home wifi and the AP is now on channel 1, away from everything else, it hasn't changed Nothing.

On the site https://dcc-ex.com/reference/hardware/wifi-boards/mega-wifi.html#gsc.tab=0 they point out that the Mega+Wifi combo cards have several quality problems, and you often run into in defective specimens. Maybe the problem is the hardware.

Does anyone have a similar setup?
Suggestions, before buying another card?
Mine has 8 Mb of memory (bought a few years ago), the ones on sale now have 16 and even 32, can the memory affect operation?

Thanks bye.
Claudio.

**Comment by flash62au** [comment]
Claudio,
I would suggest that you join our Discord server and raise the issue there.  It will be much easier to assist you.
https://discord.gg/JcXh7j3u

There a many people successfully using the Mega+WiFi, but as you mention, some have been found to be faulty.
Once you are on Discord we can run through some possible options to discover where the problem lies.

Peter
